core/templates/dev/head/domain/topics_and_skills_dashboard/TopicsAndSkillsDashboardBackendApiService.js

```diff
@@ -16,15 +16,27 @@
  * @fileoverview Service to retrieve information of topics and skills dashboard
   from the backend.
  */
+ oppia.constant(
+   'MERGE_SKILL_URL', '/merge_skill');
```

Comment:
Can we move this to the line above? (Following max 80 chars in aline.)

Reply:
Done.